packages/eds-core-react/src/components/Menu/MenuItem.tsx

```diff
@@ -78,7 +75,7 @@ const Item = styled.button.attrs<StyleAttrsProps>(({ $isFocused }) => ({
               background: ${hover.background};
             }
           }
-          &:focus {
+          &:focus-visible {
             z-index: 3;
```

Comment:
Do you know why we need this?

Reply:
This is needed so the focus.ring does not clip below the next item. However the z-index: 1 above in hover is no longer needed so i removed that one